scripts/docker/debian/sid/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apt-get update && apt-get -y install \
     autotools-dev \
     automake \
     cmake \
+    libcurl3 \
```

Comment:
I don't think all mirrors are updated yet and thus some builds for sid are currently failing (go team debian).

However I don't think explicitly requiring it here is the way to go.

Reply:
to me it just looked like the quickest solution without much hassling around. But if you think it should work without it, i can remove it again.

Reply:
I had the same error when testing the gtest changes an hour ago and a rerun of the buildjob made it pass (hence I assumed faulty mirrors).